Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM node:12-alpine as dependency-builder
 
+# add build tools for other architectures
+# subsequent builds should cache this layer
+RUN apk add make g++ python
```

Comment:
Are you sure this is required? I tried building the container on my machine without it and everything went fine. Maybe, I am overlooking something?

Reply:
AH I think I got it:

SQLite has no prebuilt binaries for some architectures, so it tries building it from source, ut it fails?

Reply:
Yes, exactly. I couldn't use the debian node image for the builder because they don't have images for some of the architectures coming in the docker multi-arch stuff. Alpine is available for everything.

Reply:
ok thanks for the clarification 😇